src/navigation/stackNavigator.js

```diff
@@ -33,6 +35,8 @@ const ChallengesStackNavigator = () => {
   return (
     <Stack.Navigator screenOptions={screenOptionStyle}>
       <Stack.Screen name="ChallengesScreen" component={Challenges} options={getNavigatorStyles('Challenges')} />
+      <Stack.Screen name="CreateChallenge" component={CreateChallenge} options={getNavigatorStyles('Create Challenge')} />
+      <Stack.Screen name="ChallengeOverview" component={ChallengeOverview} options={getNavigatorStyles('Challenge Overview')} />

```

Comment:
Lets use constants for these. Replace the name strings with path constants. Define in commonConstants.js. 
Check how reminder paths are defined.

Reply:
done